lib/entities.ts

```diff
@@ -32,6 +36,192 @@ export interface Account {
   transfers_blocked: boolean
 }
 
+/**
+ * The following are the possible account status values. Most likely, the account status
+ * is ACTIVE unless there is any problem. The account status may get in ACCOUNT_UPDATED
+ * when personal information is being updated from the dashboard, in which case you may
+ * not be allowed trading for a short period of time until the change is approved.
+ */
```

Comment:
I'm adding tons of jsdoc comments so that you get nice info in your IDE.

<img width="907" alt="Screen Shot 2020-09-11 at 3 29 28 PM" src="https://user-images.githubusercontent.com/704472/92974496-8fc10800-f443-11ea-87dc-aeb156a686a0.png">